src/include/daos/tests_lib.h

```diff
@@ -514,7 +514,7 @@ int dmg_storage_set_nvme_fault(const char *dmg_config_file,
  * Get NVMe Device health stats.
  *
  * \param[in] dmg_config_file	DMG config file
- * \param[in] host		Get device-health from the given host.
+ * \param[in] host		Get list-devices --health from the given host.
```

Comment:
Just noticed this mistaken change, looks like a find/replace change. I guess this must have been in the master PR too.

Reply:
fixed this and will fix master